src/components/action-list/ActionList.jsx

```diff
@@ -8,9 +8,14 @@ export const DIRECTION = {
   SPACE_BETWEEN: 'space-between'
 };
 
-const ActionList = ({children, toTop, direction, noWrap, className, ...props}) => {
+export const ALIGNMENT = {
+  BASELINE: 'baseline'
```

Comment:
here should be also      `align-items: flex-start` and others. So the best way will be refactor component and delete `toTop` prop. Because you can't use baseline and toTop in same time